docs/stories/v2.1/sprint-12/story-wis-5-pattern-capture.md

```
@@ -0,0 +1,461 @@
+  1. validate-story-draft → develop → review-qa (12 occurrences, 95% success)
+  2. develop → review-qa → apply-qa-fixes (8 occurrences, 88% success)
+  3. create-story → validate-story-draft → develop (6 occurrences, 100% success)
+
+  Stats:
+  - Total patterns: 15
+  - Active: 12
+  - Promoted: 3
+  - Avg success rate: 92%
+  ```
+- [ ] Review workflow for `*patterns review`:
+  ```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)